src/lib/utilities/PatientCount.ts

```diff
@@ -0,0 +1,34 @@
+import type { PatientCount, CountMap } from '$lib/types';
+
+const PLUS_MINUS = '\u00B1';
+
+function parseEntry(entry: PatientCount): { value: number; suffix: number } {
+  const [rawValue, rawSuffix] = String(entry).split(PLUS_MINUS);
+  const value = parseInt(rawValue.replace(',', '')) || 0;
+  const suffix = parseInt(rawSuffix ?? '') || 0;
+  return { value, suffix };
+}
+
+function normalizeEntries(input: CountMap | (PatientCount | undefined)[]): PatientCount[] {
+  const values = Array.isArray(input) ? input : Object.values(input);
+  return values.filter((x): x is PatientCount => x !== undefined);
+}
+
+export function countResult(
+  result: CountMap | (PatientCount | undefined)[],
+  asString = true,
+): PatientCount {
+  const entries = normalizeEntries(result);
```

Comment:
Can this var be renamed to reflect what it is in the context of the application?